lib/api/uikit/functions.php

```diff
@@ -72,8 +72,11 @@ function beans_uikit_enqueue_components( $components, $type = 'core', $autoload
 function beans_uikit_dequeue_components( $components, $type = 'core' ) {
```

Comment:
I can comment directly on Line 67.
`to exclude all components` or `excludes all components`